pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -62,13 +62,22 @@ class Build {
         if (matcher.matches()) {
             return Integer.parseInt(matcher.group('version'))
         } else if ("jdk".equalsIgnoreCase(javaToBuild.trim())) {
-            String javaFeatureVersion = System.getenv("JAVA_FEATURE_VERSION")
-            if (javaFeatureVersion) {
-                return Integer.valueOf(javaFeatureVersion)
-            } else {
-                context.error("Environment variable JAVA_FEATURE_VERSION not set")
+            // Query the Adopt api to get the "most_recent_feature_release" (currently 15)
```

Comment:
This seems wrong, would this not mean that all builds would run tests on 15/the latest version?

Reply:
Yes if the tests were enabled. We don't run tests for JDK-Next currently so there isn't any history for what would happen if we did run tests with this but it should be fine as they will expect JDK-Next binaries to be whatever the latest jdk-head number is (unless we only want to work with GA releases as described here https://github.com/AdoptOpenJDK/openjdk-build/pull/1822#issuecomment-638767984)